bin/normalizeEnv.js

```diff
@@ -8,10 +8,22 @@ const generateSecret = (secret, name) => {
   return crypto.createHmac('sha256', secret).update(name).digest('base64')
 }
 
-const getVar = (name, def = false) => {
+const getVar = (name, def = '') => {
   return process.env[name] || envVars[name] || def
 }
 
+const parseBool = (name, def = false) => {
+  const val = getVar(name).toLowerCase()
+  if (val === 'true') {
+    return true
+  } else if (val === 'false') {
+    return false
+  } else if (val === '') { // if the varialbe is undefined
+    return def
+  }
+  throw {_error: 'Can not parse as boolean: ' + name}
```

Comment:
This should trigger `process.exit(1)`, like it does [here](https://github.com/interledgerjs/ilp-kit/pull/168/files#diff-3641b3524271ca689c53abb4d4a7e8a4R55)